### ISSUE TITLE
Too many requests are started in parallel

### ISSUE THREAD

**Issue by benoit74** [openzim/mwoffliner#2092]
In https://farm.openzim.org/pipeline/53d49603-fd60-4933-b7f3-5b8130b29805/debug, the task is finally aborted due to skew between request time and server time.

What I see in the logs are many message like `@smithy/node-http-handler:WARN - socket usage at capacity=50 and 115 additional requests are enqueued.`.

To me this is the root cause of the problem, and it is a software bug. I understand this message as the software has started many HTTP requests and they are getting delayed. And hence the skew time between request time and server time from my PoV. Classic async issue in Node.JS from my PoV, where one can easily get bitten by all this asynchronous logic which has many advantages but some drawbacks.

And of course this problem occur randomly since it depends on your connectivity with the upstream. Travis connectivity to AWS is probably better than the connectivity from mwoffliner1 / wikimedia foundation datacenter.

I'll open another issue to track this problem.
